docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -99,6 +99,19 @@ imageio.mimsave("video.mp4", result, fps=4)
 ```
 
 
+- #### SDXL Support
+In order to use the SDXL model when generating a video from prompt, use the `TextToVideoZeroSDXLPipeline` pipeline:
+
+```python
+import torch
+from diffusers import TextToVideoZeroSDXLPipeline
+
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+pipe = TextToVideoZeroSDXLPipeline.from_pretrained(
+    model_id, torch_dtype=torch.float16, variant="fp16", use_safetensors=True
```

Comment:
Does this work out of the box?

Reply:
Hmm good catch. @vahramtadevosyan Can you confirm this works out of the box with SDXL base?

Reply:
@DN6 @patrickvonplaten can you explain what do you mean by `working out of the box`?